README.md

```diff
@@ -47,11 +47,13 @@
       client_secret: {:system, "GITHUB_CLIENT_SECRET"}
     ```
 
-1.  Include the Überauth plug in your controller:
+1.  Include the Überauth plug in your router:
 
     ```elixir
-    defmodule MyApp.AuthController do
-      use MyApp.Web, :controller
+    defmodule MyApp.Router do
+      use MyApp.Web, :router
+
+      require Ueberauth
```

Comment:
I don't believe this require is correct. I don't need this in any of my projects. 
```suggestion
```